scripts/tests/relayed-dns-etc-resolvconf.sh

```diff
@@ -5,21 +5,12 @@
 
 set -euo pipefail
 
-HTTPBIN=dns.httpbin
-
-function client() {
-    docker compose exec -it client "$@"
-}
+source "./scripts/tests/lib.sh"
 
-function client_nslookup() {
-    # Skip the first 3 lines so that grep won't see the DNS server IP
-    # `tee` here copies stdout to stderr
-    client timeout 30 sh -c "nslookup $1 | tee >(cat 1>&2) | tail -n +4"
-}
+HTTPBIN=dns.httpbin
 
-function gateway() {
-    docker compose exec -it gateway "$@"
-}
+install_iptables_drop_rules
+trap remove_iptables_drop_rules EXIT # Cleanup after us
```

Comment:
I wonder if the trap can go in `lib.sh` too, or is that too much?

Reply:
I'll give it a shot

Reply:
Done